src/containers/Login/component.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import {
+  Button, Col, Form, Input, Layout, Row, Typography, Alert
+} from 'antd'
+import { UserOutlined, LockOutlined } from '@ant-design/icons'
+import PropTypes from 'prop-types'
+
+const Login = ({
+  login, errorTxt, loading
+}) => (
+  <div className="center">
+    <Layout>
+      <Layout.Content>
+        <Row
+          type="flex"
+          justify="center"
+        >
+          <Col
+            sm={{ span: 10, offset: 1 }}
+            lg={{ span: 8, offset: 2 }}
+          >
+            <Form onFinish={login}>
+              <Typography.Title>The Movie DB</Typography.Title>
+              {errorTxt && (
+              <Alert
+                type="error"
+                message={errorTxt}
+              />
```

Comment:
@VladWithCookies linter wants 2d props on a new line

Reply:
@kosovych, i mean the trouble with indentation:

do
```
              {errorTxt && (
                <Alert
                  type="error"
                  message={errorTxt}
                />
              )}
```
instead of
```
              {errorTxt && (
              <Alert
                type="error"
                message={errorTxt}
              />
              )}
              ```

Reply:
I'll skip it